atmo/jobs/forms.py

```diff
@@ -26,9 +26,20 @@ class BaseSparkJobForm(FormControlFormMixin, CachedFileModelFormMixin,
             'class': 'identifier-taken-check',
             'data-identifier-taken-check-url': reverse_lazy('jobs-identifier-taken'),
         }),
-        help_text='A brief description of the scheduled Spark job\'s purpose, '
+        help_text='A slug-like string to identify your Spark job, '
```

Comment:
Question: Does this string need to be a slug? Or can it just be a short string with spaces or capitals? I didn't see any slug processing but all the examples in tests are "slug-like".